### DISCUSSION TITLE
ADIF 2MB import  limit

### DISCUSSION THREAD

**Discussion by ae8u** [magicbug/Cloudlog#3213] [Q&A] [answered]
I am trying to set up Cloudlog. But my ADIF export out of my current logging program is over the 2MB limit. Is this a hard limit that cannot be changed, or there a config file somewhere that I can edit to increase this limit?

**Comment by magicbug** [comment]
This is a PHP configuration limit, you can increase by adjusting its config file

**Reply by ae8u** [reply]
Can you tell me what specific file and which survival it is located in?  I have tried to find it without luck.

**Reply by rayslinky** [reply, marked as answer]
There should be a php.ini somewhere in /etc. You'll need to change upload_max_filesize and possibly post_max_size
